docs/en/latest/config.json

```diff
@@ -0,0 +1,10 @@
+{
+    "version": 1.0,
```

Comment:
It seems @tzssangglass wants to make a `0.1` release:
https://github.com/apache/apisix-java-plugin-runner/pull/35#issuecomment-864721810